play/type-racer/app.js

```diff
@@ -120,11 +188,17 @@ function stopLightStart() {
 /**
  * Start the game.
  */
-function startGame() {
+async function startGame() {
     // prevent startGame() from being called multiple times before ending
     if (gameStarted) {return;}
```

Comment:
If you're going to do it in one line like this, I think it would look better to do:
```js
if (gameStarted) return;
```
However this is more of a personal thing than anything.